pkg/minikube/cluster/cluster.go

```diff
@@ -584,7 +584,7 @@ func GetServiceURLs(api libmachine.API, namespace string, t *template.Template)
 }
 
 func GetCA(h sshAble) (string, error) {
```

Comment:
I think we can remove this method completely because I didn't find this is used anywhere in the codebase now.